CHANGELOG.md

```diff
@@ -71,6 +71,8 @@ These changes are available on the `master` branch, but have not yet been releas
 - Added `suppress` and `allowed_mentions` parameters to `Webhook` and
   `InteractionResponse` edit methods.
   ([#2138](https://github.com/Pycord-Development/pycord/pull/2138))
+- Allowed more classes in `paginator.edit`.
```

Comment:
Not sure if this change counts as an addition instead of a change.

Reply:
I think added is fine

Reply:
I could change the line to `Added more classes in paginator.edit.` if that makes it better.

Reply:
yeh